appveyor.yml

```diff
@@ -21,7 +21,7 @@ environment:
       # are in astropy-ci-extras. If your package uses either of these,
       # add the channels to CONDA_CHANNELS along with any other channels
       # you want to use.
-      # CONDA_CHANNELS: "astopy-ci-extras astropy"
+      CONDA_CHANNELS: "astropy-ci-extras astropy"
```

Comment:
Also don't remove the comments here, most appveyor runs are not testing against LTS, and in many cases they don't use affiliated packages from the astropy channel either.